src/main/kotlin/com/back/domain/auth/service/AuthService.kt

```
@@ -0,0 +1,123 @@
+
+            // 3. JWT 생성
+            val accessToken = jwtTokenProvider.generateAccessToken(member)
+            val refreshToken = jwtTokenProvider.generateRefreshToken(member)
+
+            // 4. 리프레시 토큰 저장
+            member.updateRefreshToken(refreshToken)
+            saveMemberSafely(member, "토큰 저장에 실패했습니다.")
+
+            // 5. DTO 응답 반환
+            MemberLoginResponse(accessToken, refreshToken, fromEntity(member))
+        } catch (e: BadCredentialsException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)